std_msgs/String project_json
std_msgs/String default_settings
visualization_msgs/MarkerArray current_path